app/updater.js

```diff
@@ -131,17 +131,16 @@ var requestVersionInfo = (done, pingOnly) => {
   if (!platformBaseUrl) throw new Error('platformBaseUrl not set')
 
   // Get the daily, week of year and month update checks
-  var lastCheckYMD = AppStore.getState().toJS().updates['lastCheckYMD'] || null
+  const state = AppStore.getState()
+  var lastCheckYMD = state.getIn(['updates', 'lastCheckYMD']) || null
```

Comment:
Could you update this to use the default value format? ex:
`var lastCheckYMD = state.getIn(['updates', 'lastCheckYMD'], null)`

might as well make the variables `const` too instead of `var` 😄